src/sip128.rs

```diff
@@ -496,7 +502,11 @@ impl Hasher for SipHasher128 {
     }
 
     fn finish(&self) -> u64 {
-        panic!("SipHasher128 cannot provide valid 64 bit hashes")
+        let mut buf = self.buf.clone();
```

Comment:
Seems like we still need to clone the buffer 🤷
`Hasher::finish` is just defined in an unfortunate way.